readme.md

```diff
@@ -427,7 +427,6 @@
 | [W3Layouts](https://w3layouts.com/)| W3Layouts: 3784+ Free Website Templates for 2020 |
 | [Tooplate](https://www.tooplate.com/)| Tooplate: Free HTML Templates for everyone! |
 | [Cruip](https://cruip.com/free-templates/)| Fully coded HTML templates to help you easily build your startup landing page without hassles. |
-| [Shade](https://grayic.com/shade/) | Get High Converting Landing Pages For Free! |
```

Comment:
Looks like the website is under construction, let's give it another month perhaps?

Reply:
Sure, no problem. We can just leave the PR if you want. If it's still not online by the end of the year maybe it's time to merge it.

Reply:
🚀

Reply:
Website is still down. I vote for removal.